### PR TITLE
Support for data operations on all data (Now with user prompt as requested)

### PR DESCRIPTION
**Pull request by datasynergyuk**
Corrected grammar. To "use" hex is probably better than "put" (which doesn't make so much sense)

Please can I suggest not shrinking operations dialog layout just yet as I have more operations to suggest in next PR

### CODE REVIEW

**Comment by jovibor** [comment]
Few notes.
1. If you make any selection then open Oper dialog, you see both radio available `All` and `Selection`. Now if you click back into HexCtrl (Opers is still opened) and remove selection, and click back to Opers the `Selection` radio is still active, in my understanding it must be disabled at this stage.
2. Unrelated to the PR itself. 
It's about formatting, please make sure that the `Follow project coding conventions` switch is checked in VS properties. It's important that the code formatting style remains the same across all developers, so there is no commit diffs just because of VS code reformattings.
image


> Corrected grammar. To "use" hex is probably better than "put" (which doesn't make so much sense)

Well, English is not my native language and you must know better, but what actually is wrong with the 'put' here, could you please shed some light? In my understanding it's quite correct sentence, isn't it?
Basically I just didn't like doubling the 'use' here that's why this change, but it's not a big of a deal.

**Comment by datasynergyuk** [comment]
Thank you for the feedback.

1. I have corrected the radio button issue.
2. I already have the "Follow project coding conventions" switch enabled. I think the problem may be that I hand edited the .RC file? Please, which file(s) broke the expected style? I try to avoid unnecessary changes in PRs so that my intent is clear.
3. I don't think "put" is strictly wrong but, to my ears, it certainly sounds more clumsy that "use". I agree that "use" twice in sentence may look awkward but this didn't stand out to me when I read it. Splitting hairs, the same sentence could be written better with a comma. The following are all roughly the same:

To use Hex values, use 0x as prefix:
To use Hex values, use 0x prefix:
For hex values, use 0x prefix:
For hex values, put 0x prefix:     = Probably the most clumsy feel
